<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALCULADORA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 170, 0);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="valor1">
       <property name="text">
        <string>Valor 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ivalor1"/>
     </item>
     <item>
      <widget class="QLabel" name="valor2">
       <property name="text">
        <string>Valor 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ivalor2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnbotao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>CALCULAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="total">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>111</width>
      <height>118</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="CBsomar">
       <property name="text">
        <string>Somar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CBsubtrair">
       <property name="text">
        <string>Subtrair</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CBmultiplicar_2">
       <property name="text">
        <string>Multiplicar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CBdividir">
       <property name="text">
        <string>Dividir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
